CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Changelog
 =========
 
```

Comment:
i have seen the thing with "x.y.0 (unreleased)" in other projects and
found it quite useful. it will come in handy when we merge 1.3 to master
so that we don't confuse this section with the 2.0 changes section.